code/game/objects/items/clown_items.dm

```diff
@@ -148,6 +148,7 @@
 	hitsound = null //To prevent tap.ogg playing, as the item lacks of force
 	w_class = WEIGHT_CLASS_TINY
 	slot_flags = ITEM_SLOT_BACK|ITEM_SLOT_BELT
+	item_flags = ISWEAPON
```

Comment:
Did you make sure you are not overriding any flag from item?

Reply:
same with toys

Reply:
I went through the list, spawned the items in and checked them through VV, none of them had any item flags.
Well apart from the toy dualsaber, which had the 'ISWEAPON' flag passed down from the actual double esword. Should I remove it from the toy dualsaber then?

Reply:
Arch was asking if you were accidentally removing unrelated `item_flags` from the parent types

Reply:
I did check it by spawning the items and seeing their flags with the VV menu, it's not overwrotong anything.

Reply:
I'll check in code later to confirm this 👍